src/journal/journald-server.c

```diff
@@ -82,48 +82,27 @@
 
 #define NOTIFY_SNDBUF_SIZE (8*1024*1024)
 
-static int determine_space_for(
-                Server *s,
-                JournalMetrics *metrics,
-                const char *path,
-                const char *name,
-                bool verbose,
-                bool patch_min_use,
-                uint64_t *available,
-                uint64_t *limit) {
 
-        uint64_t sum = 0, ss_avail, avail;
+static int determine_path_usage(Server *s, const char *path, uint64_t *used, uint64_t *free) {
```

Comment:
So iiuc this is supposed to make no functional changes, just splits out some code into a function of its own? Could you say that explicitly in the commit msg so that one doesn't have to guess?